lib/services/elasticsearch.js

```diff
@@ -994,8 +994,8 @@ function getElasticsearchRequest(request, kuzzle, extraParams = []) {
   const data = {};
 
   if (request.input.resource.index) {
-    if (request.input.resource.index === kuzzle.internalEngine.index) {
-      throw new BadRequestError(`Cannot operate on Kuzzle internal index "${kuzzle.internalEngine.index}"`);
+    if (request.input.resource.index[0] === '%') {
+      throw new BadRequestError(`Cannot operate on Kuzzle internal index "${request.input.resource.index}"`);
```

Comment:
I would change this error message to make it more explicit for the client.

Proposal: `Indexes starting with a "%" are reserved for internal use. Cannot process index ${request.input.resource.index}`